### PR TITLE
Smaller logo and several improvements

### PR DESCRIPTION
**Pull request by theodorosploumis**
Solving #10 and providing better css.

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -25,7 +25,7 @@ h1 {
   background: -o-radial-gradient(#915f6d66, #915f6d); /* For Opera 11.6 to 12.0 */
   background: -moz-radial-gradient(#915f6d66, #915f6d); /* For Firefox 3.6 to 15 */
   background: radial-gradient(#915f6d66, #915f6d); /* Standard syntax */
-  color: #fff;
+  color: #384452;
```
Comment (CuriousLearner) [inline assets/css/main.css]: White color looks good to me in this. 

@shivan1b review?

**Review by CuriousLearner — COMMENTED** [review]
Apart from the color change, rest looks good to me.

Waiting for @shivan1b's review.

**Comment by theodorosploumis** [comment]
I did the CSS change because the **color contrast** was obvious not suitable. Test with http://www.checkmycolours.com/ to find out what I mean.

**Comment by CuriousLearner** [comment]
Hi @theodorosploumis! Can you please rebase your changes?

File: assets/css/main.css [hunk 2]
```diff
@@ -20,12 +20,12 @@ h1 {
 }
 
 .jumbotron {
-  background: #ff407c; /* For browsers that do not support gradients */
-  background: -webkit-radial-gradient(#915f6d, #915f6d); /* Safari 5.1 to 6.0 */
-  background: -o-radial-gradient(#915f6d, #915f6d); /* For Opera 11.6 to 12.0 */
-  background: -moz-radial-gradient(#915f6d, #915f6d); /* For Firefox 3.6 to 15 */
-  background: radial-gradient(#915f6d, #915f6d); /* Standard syntax */
-  color: #fff;
+  background: #ff407ce6; /* For browsers that do not support gradients */
```
Comment (CuriousLearner) [inline assets/css/main.css]: These issues were solved in PR #12 

Can you please rebase your branch with current master to have these changes?

**Review by CuriousLearner — APPROVED** [review]
Looks good to me!

Thanks for your contributions!

**Pull request merged**